### ISSUE TITLE
Use different `aria-label` strings on Space landing page

### ISSUE THREAD

**Issue by luixxiul** [vector-im/element-web#25119]
#### What would you like to do?

On Space landing page, the ARIA attribute `aria-label="Space"` can be found on multiple elements (on the left panel and on the Spaces list). Each should be specified with different strings.

#### Why would you like to do it?

To make those elements more accessible.

**Comment by luixxiul** [comment]
Now I am not sure if using different strings is sensible.
